### ISSUE TITLE
enum values remove underscores

### ISSUE THREAD

**Issue by mikestaub** [IBM/openapi-to-graphql#273]
The spec we are wrapping has enum fields that have underscores, but the generated graphQL schema has them removed. This is causing the queries to break, is there a way to disable this behavior?

**Comment by Alan-Cha** [comment]
@mikestaub Thanks for filling this issue. This must be a bug. I know you reached out earlier about another issue regarding enums and even created a sample repository. Unfortunately, I haven't had a chance to look at that issue either. Let me take a look now and see if I can give you more insight into the problem. Thanks for your patience.

**Comment by Alan-Cha** [comment]
@mikestaub I was able to follow up on #266 and I wonder if the issues are related. Could you let me know if these enum fields correspond to other GraphQL components? I wonder if this is related to the problematic desanitization that I mentioned in the other issue.

**Comment by mikestaub** [comment]
These enum fields are specific to just one type and are not shared. I don't think the issues are related.

**Comment by Alan-Cha** [comment]
@mikestaub Is it possible to share the OAS (or a portion of it) so I can try to debug?

**Comment by mikestaub** [comment]
Yes, here is a simplified example: 
```
swagger: "2.0"
info:
  version: "1.0.0"
  title: Test API

host: localhost:1339
basePath: /

schemes:
  - http
  - https

consumes:
  - application/json
produces:
  - application/json

x-servers:
  - url: //api.test.com
    description: Default server

paths:
  /v1/files:
    post:
      summary: Create a file
      operationId: postFile
      description: Creates a new file
      responses:
        "201":
          description: Success
          schema:
            $ref: "#/definitions/File"

definitions:
  Type:
    enum:
      - TEST_ENUM
    type: string

  File:
    type: object
    properties:
      cadType:
        $ref: "#/definitions/Type"

```

**Comment by Alan-Cha** [comment]
I working on a fix #277. I have the desired behavior but it is breaking a few other tests which I'm still trying to figure out.
